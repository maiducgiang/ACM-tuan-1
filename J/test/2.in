3
89,73(14)
39,84(15)
62,9(77)
