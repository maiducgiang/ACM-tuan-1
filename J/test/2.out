888341 9900
131477 3300
2834 45
